### PR TITLE
feat(perf): enable batched swa

### PR DESCRIPTION
**Pull request by rebel-jaehunryu**
### 🚀 Summary of Changes

> *What does this PR do? What feature, fix, or improvement does it bring?*

---

### 📌 Related Issues / Tickets

* Resolves #
* Related to #

---

### ✅ Type of Change

* [ ] ✨ Feature (`feature`)
* [ ] 🧠 Model support (`model`)
* [ ] 🧬 Core engine changes (`core`)
* [ ] 🛠 Bug fix (`bug-fix`)
* [ ] ⚙️ Performance improvement (`perf`)
* [ ] 🔁 Refactor or code cleanup (`refactor`)
* [ ] 📄 Documentation (`docs`)
* [ ] ❓ Other (`other`): please describe

---

### 🧪 How to Test

1. Run `...`
2. Verify output: `...`
3. Edge case tested: `...`

---

### 📸 Screenshots / Logs (if applicable)

---

### 📋 Checklist

* [ ] PR title follows Conventional Commits format
* [ ] This PR is linked to an existing issue
* [ ] The test method is described, and the expected result is clearly stated
* [ ] Relevant documentation has been updated (if applicable)

---

### 💬 Notes

---

### CODE REVIEW

File: vllm_rbln/v1/attention/backends/flash_attention.py [hunk 1]
```diff
@@ -707,13 +708,7 @@ def sliding_window_attention_naive_decode_impl(
...
 
         attn_weights = torch.matmul(q_r, k_cache_curr.transpose(3, 4)) * scale
-
-        ones = torch.ones(window_size + 1, window_size + 1)
-        mask_full = torch.tril(ones) - torch.tril(ones, diagonal=-window_size)
-        mask = mask_full[None, None, None, cache_start:cache_start + 1, :]
-        mask = torch.where(mask > 0, 0.0, float('-inf')).to(attn_weights.dtype)
-
-        attn_weights = attn_weights + mask
+        attn_weights = attn_weights + attn_mask
```
Comment (rebel-jaehwang) [inline vllm_rbln/v1/attention/backends/flash_attention.py]: is the new mask compatible with this cpu implementation?

File: vllm_rbln/v1/attention/backends/flash_attention.py [hunk 2]
```diff
@@ -1012,6 +1015,12 @@ def build(
             if not is_prefills[0]:
                 cache_seq_lens = rbln_utils.pad(cache_seq_lens, 0, batch_pad)
                 cache_offsets = rbln_utils.pad(cache_offsets, 0, batch_pad)
+                # Generate sliding window attention mask for decode
+                # mask[b, s] = 1.0 if s <= cache_seq_lens[b] else 0.0
+                positions = torch.arange(sliding_window)[None, :]
+                swa_attn_masks = torch.where(
```
Comment (rebel-jaehunryu) [inline vllm_rbln/v1/attention/backends/flash_attention.py]: https://github.com/RBLN-SW/optimum-rbln/blob/93ad3a0fd3b51d1f3d56c9355dc621687d780b1e/src/optimum/rbln/transformers/models/decoderonly/decoderonly_architecture.py#L389-L401

**Review by rebel-jaehwang — APPROVED** [review]
